.github/workflows/snyk-scan-pr.yml

```diff
@@ -0,0 +1,41 @@
+---
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Contributors to the OpenEXR Project.
+
+name: Snyk Scan Code
+
+on:
+  # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+  pull_request:
+    branches:
+      - main
```

Comment:
Could you add a "paths:"  filter here so it triggers only on changes to .cpp, .c, or .h files? The repo has documentation/website source that shouldn't trigger an analysis.

Reply:
Done!